game/static/game/js/game.js

```diff
@@ -500,7 +500,7 @@ ocargo.Game.prototype._setupTabs = function () {
     this._setupQuitTab();
     this._setupNightModeTab();
 
-    if (USER_STATUS === 'TEACHER'){
+    if (USER_STATUS === 'TEACHER' && LEVEL_NAME <= 109){
```

Comment:
I'd rather use the default field of the Level model :) True = Default level, False = Custom level - this way we can add more default levels without changing this piece of code!